jackbot.py

```diff
@@ -68,9 +66,12 @@ async def _getRoles(self):
             roleshash[role.name] = role
         return roleshash
     
-    def _generateMsg(self, merchant, role_name):
-        msg = "{} is selling {} in {} - ".format(merchant.name, role_name, merchant.region)
-        msg += "[{}]({})".format(merchant.zone, URL + merchant.zone_img)
+    def _generateMsg(self, merchant, embedLink=False):
+        msg = f"{merchant.name} is selling <@&{roleshash[merchant.card].id}> in {merchant.region} - "

```

Comment:
roleshash is undefined in this function

Reply:
Should be good now: https://github.com/mayo320/jackbot/pull/1/files#diff-c3c4e3c1460ac3e98fa32e00e9526a448fe1a4ac77d92066ff3a55656169daddR68